x-pack/plugins/ml/public/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -89,7 +91,7 @@ function getSvgHeight() {
   return focusHeight + contextChartHeight + swimlaneHeight + chartSpacing + margin.top + margin.bottom;
 }
 
-export class TimeseriesChart extends React.Component {
+export const TimeseriesChart = injectI18n(class TimeseriesChart extends React.Component {
```

Comment:
The zoom options are not being localized. Not sure if we are translating the Elasticsearch intervals `1h, 12h, 1w` etc, but I'm thinking `auto` should definitely be translated. The labels are defined in this file.

![image](https://user-images.githubusercontent.com/7405507/50967338-c8227d00-14cf-11e9-85cd-9386534dec3f.png)

Reply:
These **intervals** are in the [Glossary](https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w)(<code>Date interval, e.g. auto, 1m, 1d, 7d, 1y, >=1m</code>), and we don't need to  translate them.

Reply:
OK!